### ISSUE TITLE
[CCXT] UnboundLocalError: local variable 'freqtrade' referenced before assignment

### ISSUE THREAD

**Issue by xFFFFF** [gcarq/freqtrade#615]
## Step 1: Have you search for this issue before posting it?

If you have discovered a bug in the bot, please [search our issue tracker](https://github.com/gcarq/freqtrade/issues?q=is%3Aissue). 
If it hasn't been reported, please create a new issue.

## Step 2: Describe your environment

  * Python Version: 3.6(`python -V`)
  * Branch: objetify-ccxt
  * Last Commit ID: 78bafee39db93e603d1fc86fe2ed7f420c8c1bc0 (`git log --format="%H" -n 1`)
 
## Step 3: Describe the problem:
*Explain the problem you have encountered*
I start freqtrade first time. I set exchange binance, put api keys, change btc to bnb, and pairs to bnb in config.json. telegram disabled. i tried dry run enables and disabled.

### Steps to reproduce:

  1. Install by this way: https://github.com/gcarq/freqtrade/blob/develop/docs/installation.md#linux---ubuntu-1604
  2. First start by python3.6 ./freqtrade/main.py -c config.json command

### Observed Results:

  * What happened?
  * What did you expect to happen?

### Relevant code exceptions or logs:
```
python3.6 ./freqtrade/main.py -c config.json
2018-04-21 01:37:20,091 - root - INFO - Generating grammar tables from /usr/lib/python3.6/lib2to3/Grammar.txt
2018-04-21 01:37:20,130 - root - INFO - Generating grammar tables from /usr/lib/python3.6/lib2to3/PatternGrammar.txt
2018-04-21 01:37:20,293 - freqtrade - INFO - Starting freqtrade 0.16.0 (loglevel=INFO)
2018-04-21 01:37:20,294 - freqtrade.configuration - INFO - Using config: config.json ...
2018-04-21 01:37:20,294 - freqtrade - ERROR - Fatal exception!
Traceback (most recent call last):
  File "./freqtrade/main.py", line 48, in main
    freqtrade = FreqtradeBot(configuration.get_config())
  File "/home/f/freqtrade/freqtrade/configuration.py", line 202, in get_config
    self.config = self.load_config()
  File "/home/f/freqtrade/freqtrade/configuration.py", line 34, in load_config
    config = self._load_config_file(self.args.config)
  File "/home/f/freqtrade/freqtrade/configuration.py", line 58, in _load_config_file
    conf = json.load(file)
  File "/usr/lib/python3.6/json/__init__.py", line 299, in load
    parse_constant=parse_constant, object_pairs_hook=object_pairs_hook, **kw)
  File "/usr/lib/python3.6/json/__init__.py", line 354, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python3.6/json/decoder.py", line 339, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python3.6/json/decoder.py", line 357, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 20 column 9 (char 551)
Traceback (most recent call last):
  File "./freqtrade/main.py", line 74, in <module>
    main(sys.argv[1:])
  File "./freqtrade/main.py", line 59, in main
    freqtrade.clean()
UnboundLocalError: local variable 'freqtrade' referenced before assignment
```

**Comment by xmatthias** [comment]
The initial error seems to be an error in your configuration:
`json.decoder.JSONDecodeError: Expecting value: line 20 column 9 (char 551)`

The bug regarding the "referenced before assignment" was fixed in 9d443b8bd837cdc7d66abeb585b5d70927f08332 - 
we should probably cherry-pick that one to ccxt to simplify testing - @shusso what do you think?

**Comment by xFFFFF** [comment]
Fixed. Thanks.
